src/PowerManifold.jl

```diff
@@ -376,6 +376,9 @@ functions defined for the power manifold, meaning that this is used elementwise.
 function default_retraction_method(M::PowerManifold)
     return default_retraction_method(M.manifold)
 end
+function default_retraction_method(M::PowerManifold, t::Type)
+    return default_retraction_method(M.manifold, t)
```

Comment:
```suggestion
    return default_retraction_method(M.manifold, eltype(t))
```
At least for nested representations this should pass `eltype`.

Reply:
Similarly for other power manifold methods.

Reply:
Oh, well spotted :) I was not even aware that `eltype` works on array types as well.